requirements.functest.txt

```diff
@@ -13,7 +13,6 @@ git+https://github.com/F5Networks/pytest-symbols.git
 mock==2.0.0
 pytest>=3.0.5,<4
 pytest-cov>=2.4.0,<3
-responses==0.5.1
```

Comment:
By inspecting with "grep -r" I formed the belief that this package is never used.